docker/ubuntu/config.yaml

```diff
@@ -17,4 +17,5 @@ base_config: |
         libxml2-dev \
         libxpm-dev \
         libxt-dev \
-        texinfo
+        texinfo \
+        libgccjit-5-dev gcc-5 g++-5
```

Comment:
We are using `libgccjit-9-dev`,  `gcc-9`,  `g++-9` at build.

https://github.com/emacs-ng/emacs-ng/blob/bc889e9eebcc01937e05d559594cde1027cef479/build_emacs_ng.sh#L19

Is there any reason to use the lower version?

Reply:
@black7375 this is the GCC version availabe in Ubuntu Xenial (which this image is based on).

To be honest I don't have clear why emacs-ng needs these Docker images at all (based on ancient Ubuntu or 32bit Ubuntu versions)

Reply:
It could be because it was made a long time ago